Server/models/Metrics.js

```diff
@@ -0,0 +1,52 @@
+const mongoose = require("mongoose");
+
+/**
+ * Mongoose schema for storing metrics from Google Lighthouse.
+ * @typedef {Object} MetricsSchema
+ * @property {mongoose.Schema.Types.ObjectId} monitorId - Reference to the Monitor model.
+ * @property {Object} accessibility - Metrics related to accessibility category.
+ * @property {Object} bestPractices - Metrics related to best practices category.
+ * @property {Object} seo - Metrics related to SEO category.
+ * @property {Object} performance - Metrics related to performance category.
+ * @property {Date} timestamp - Timestamp when the metrics were recorded.
+ */
+
+const MetricsSchema = mongoose.Schema(
+  {
+    monitorId: {
+      type: mongoose.Schema.Types.ObjectId,
+      ref: "Monitor",
+      immutable: true,
+    },
+    accessibility: {
+      type: Object,
```

Comment:
These can all be type number as all were interested in from these (for now) is the overall score. If we need more metrics we can create more fields later